edx_exams/apps/api/serializers.py

```diff
@@ -41,3 +41,13 @@ def validate_exam_type(self, value):
         if value not in valid_exam_types:
             raise serializers.ValidationError("Must be a valid exam type.")
         return value
+
+
+class ProctoringProviderSerializer(serializers.ModelSerializer):
```

Comment:
I was thinking a little bit about this. Is there a chance we're going to add any fields to the `ProctoringProvider` model that we _don't_ want to be publicly accessible? I don't think we've figured out where or how we're storing configuration data for providers. If so, we'll want to explicitly call out what fields we want to be returned by the API in this serializer instead of "all" so we don't inadvertently leak sensitive data in the future.

CC: @Zacharis278

Reply:
I don't foresee anything in this table being sensitive since this is mostly just an intermediate between an exam and an LTIConfiguration (which would be sensitive).  That's said I think you have a point here, and it's overall safer to enumerate the fields we're returning even if it is all of them.

Could save an issue of my assumption changes in the future

Reply:
got it, this makes sense